blivet/nvdimm.py

```diff
@@ -0,0 +1,129 @@
+#
+# nvdimm.py - nvdimm class
+#
+# Copyright (C) 2018  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import gi
+gi.require_version("BlockDev", "2.0")
+from gi.repository import BlockDev
+
+import logging
+log = logging.getLogger("blivet")
+
+
+class NVDIMM(object):
```

Comment:
Can this go into `blivet.static_data` instead? I was hoping to move `iscsi` and `fcoe` there eventually as well.

Reply:
Yes, I'll move it.

Reply:
Moved.